### PR TITLE
ui v2: tabs

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Introduce tab & tabs components

tabs

### Testing Performed

Storybook

### CODE REVIEW

File: frontend/packages/core/src/tab.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import styled from "@emotion/styled";
+import type { TabProps as MuiTabProps, TabsProps as MuiTabsProps } from "@material-ui/core";
+import { Tab as MuiTab, Tabs as MuiTabs } from "@material-ui/core";
+
+const StyledTab = styled(MuiTab)({
+  minWidth: "111px",
+  height: "46px",
+  padding: "0",
+  color: "rgba(13, 16, 48, 0.6)",
+  borderBottom: "3px solid #E7E7EA",
```
Comment (scarlettperry) [inline frontend/packages/core/src/tab.tsx:11]: it looks a little thicker on figma. i think it should be 4px
```suggestion
  borderBottom: "4px solid #E7E7EA",
```

File: frontend/packages/core/src/tab.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    color: "#3548D4",
+    backgroundColor: "#EBEDFB",
+  },
+  "&:focus-within": {
+    color: "#3548D4",
+    backgroundColor: "#EBEDFB",
+  },
+  "&:active": {
+    color: "rgba(13, 16, 48, 0.6)",
+    backgroundColor: "rgba(219, 219, 224, 1)",
```
Comment (scarlettperry) [inline frontend/packages/core/src/tab.tsx:34]:
```suggestion
    backgroundColor: "#DBDBE0",
```

**Comment by scarlettperry** [comment]
for the single tab story, noticed that the selected tab doesn't have the bottom blue border

File: frontend/packages/core/src/tab.tsx (hunk 1 above)
Comment (scarlettperry) [reply frontend/packages/core/src/tab.tsx:11]: actually nvm, it's correct. on figma it's only the active tab that has the thicker border

File: frontend/packages/core/src/tab.tsx (hunk 2 above)
Comment (dschaller) [reply frontend/packages/core/src/tab.tsx:34]: This is the rgb value :)

**Comment by dschaller** [comment]
yeah, the marker is only available on tabs. We can force the bottom border on selected but I think it doesn't make sense to since we already have it from the marker when activated.

**Pull request merged**